blocks/ai-text/src/dev.tsx

```diff
@@ -22,8 +38,14 @@ const DevApp = () => {
     <MockBlockDock
       blockDefinition={{ ReactComponent: Component }}
       blockInfo={packageJSON.blockprotocol}
+      initialData={{
+        initialEntities: [
+          testEntity,
+          // ...imageEntities
+        ],
+      }}
       // @todo add dot-env support
-      blockProtocolApiKey={undefined} // Set this to an API key when testing
+      blockProtocolApiKey="b10ck5.43430b0bbd608928f03b8fe5aff452c8.fe03fe6b-e8ed-4725-8eae-adcb890af804" // Set this to an API key when testing
```

Comment:
```suggestion
      blockProtocolApiKey={undefined} // Set this to an API key when testing
```

Reply:
Ah this file was never meant to be comitted, my bad!